cpu/Makefile.include.cortexm_common

```diff
@@ -11,6 +11,18 @@ export USEMODULE += newlib
 # set default for CPU_MODEL
 export CPU_MODEL ?= $(CPU)
 
+# Temporary LLVM/Clang Workaround:
+# report cortex-m0 instead of cortex-m0plus if llvm/clang (<= 3.6.2) is used
+# llvm/clang version 3.6.2 still does not support the cortex-m0plus mcpu type
+ifeq (llvm,$(TOOLCHAIN))
+ifeq (cortex-m0plus,$(CPU_ARCH))
+LLVM_CHECK = $(shell llvm-as < /dev/null | llc -march=$(CPU_MODEL) -mcpu=$(CPU_ARCH))
```

Comment:
I'm not sure this works. What is -march= supposed to be?
I tried with `-mcpu=cortex-m8` and I got the following output:

```
$ llvm-as < /dev/null | llc -march=thumb -mcpu=cortex-m8
'cortex-m8' is not a recognized processor for this target (ignoring processor)
'cortex-m8' is not a recognized processor for this target (ignoring processor)
'cortex-m8' is not a recognized processor for this target (ignoring processor)
'cortex-m8' is not a recognized processor for this target (ignoring processor)
'cortex-m8' is not a recognized processor for this target (ignoring processor)
'cortex-m8' is not a recognized processor for this target (ignoring processor)
'cortex-m8' is not a recognized processor for this target (ignoring processor)
'cortex-m8' is not a recognized processor for this target (ignoring processor)
        .text
        .syntax unified
        .eabi_attribute 67, "2.09"      @ Tag_conformance
'cortex-m8' is not a recognized processor for this target (ignoring processor)
'cortex-m8' is not a recognized processor for this target (ignoring processor)
'cortex-m8' is not a recognized processor for this target (ignoring processor)
'cortex-m8' is not a recognized processor for this target (ignoring processor)
'cortex-m8' is not a recognized processor for this target (ignoring processor)
'cortex-m8' is not a recognized processor for this target (ignoring processor)
        .cpu    cortex-m8
        .eabi_attribute 6, 1    @ Tag_CPU_arch
        .eabi_attribute 8, 1    @ Tag_ARM_ISA_use
        .eabi_attribute 9, 1    @ Tag_THUMB_ISA_use
        .eabi_attribute 17, 1   @ Tag_ABI_PCS_GOT_use
        .eabi_attribute 20, 1   @ Tag_ABI_FP_denormal
        .eabi_attribute 21, 1   @ Tag_ABI_FP_exceptions
        .eabi_attribute 23, 3   @ Tag_ABI_FP_number_model
        .eabi_attribute 34, 0   @ Tag_CPU_unaligned_access
        .eabi_attribute 24, 1   @ Tag_ABI_align_needed
        .eabi_attribute 25, 1   @ Tag_ABI_align_preserved
        .eabi_attribute 38, 1   @ Tag_ABI_FP_16bit_format
        .eabi_attribute 14, 0   @ Tag_ABI_PCS_R9_use
        .file   "<stdin>"

        .section        ".note.GNU-stack","",%progbits
```
and the exit code was 0

with `-mcpu=cortex-m0` I get the same minus the message about nonexistant CPU:

```
llvm-as < /dev/null | llc -march=thumb -mcpu=cortex-m0    
        .text
        .syntax unified
        .eabi_attribute 67, "2.09"      @ Tag_conformance
        .cpu    cortex-m0
        .eabi_attribute 6, 12   @ Tag_CPU_arch
        .eabi_attribute 8, 0    @ Tag_ARM_ISA_use
        .eabi_attribute 9, 1    @ Tag_THUMB_ISA_use
        .eabi_attribute 17, 1   @ Tag_ABI_PCS_GOT_use
        .eabi_attribute 20, 1   @ Tag_ABI_FP_denormal
        .eabi_attribute 21, 1   @ Tag_ABI_FP_exceptions
        .eabi_attribute 23, 3   @ Tag_ABI_FP_number_model
        .eabi_attribute 34, 0   @ Tag_CPU_unaligned_access
        .eabi_attribute 24, 1   @ Tag_ABI_align_needed
        .eabi_attribute 25, 1   @ Tag_ABI_align_preserved
        .eabi_attribute 38, 1   @ Tag_ABI_FP_16bit_format
        .eabi_attribute 14, 0   @ Tag_ABI_PCS_R9_use
        .file   "<stdin>"

        .section        ".note.GNU-stack","",%progbits
```
also with an exit code of 0